imap_processing/idex/l1/idex_l1.py

```diff
@@ -19,9 +21,64 @@
 
 logger = logging.getLogger(__name__)
 
+# TODO: Generate quicklook plots
+# TODO: Currently the code assumes that one L0 file will produce one L1 file.
+#       Is this a valid assumption?
+
+# Create a large dictionary of values from the FPGA header that need to be
+# captured into the CDF file.  They are lumped together because they share
+# similar attributes.
+# Notes about the variables are set here, acting as comments and will also be
+# placed into the CDF in the VAR_NOTES attribute.
+TRIGGER_DESCRIPTION = namedtuple(
+    "TRIGGER_DESCRIPTION",
+    ["name", "packet_name"],
+)
+TRIGGER_DESCRIPTION_DICT = {
```

Comment:
Moved this to the top of the file and made them ALL_CAPS since they are treated as global variables